.github/workflows/ci-shell-scripts.yml

```diff
@@ -0,0 +1,33 @@
+# https://help.github.com/en/categories/automating-your-workflow-with-github-actions
+
+name: "Continuous Integration for shell scripts"
+
+on:
+  pull_request:
+    paths:
+      - ".github/workflows/ci-shell-scripts.yml"
+      - "bin/**"
+  push:
+    branches:
+      - "*.*.x"
+    paths:
+      - ".github/workflows/ci-shell-scripts.yml"
+      - "bin/**"
+  schedule:
+    - cron: "0 17 * * *"
```

Comment:
:thinking: why?

Reply:
The "general" CI pipeline is executed with the same schedule. IMO, is useful to have this pipeline on the same basis, as the rules or something else at ShellCheck or the GH action could be updated without a previous notice.

Reply:
Oh right indeed… TIL!